FAQ/FAQ-Altcoin.md

```diff
@@ -20,15 +20,16 @@ Bitcoin is the only focus of the project and its core developers. However, opt i
 - Groestlcoin (GRS)
 - Litecoin (LTC)
 - Monacoin (MONA)
+- Monero (XMR) [(notes on deployment)](https://github.com/btcpayserver/btcpayserver-docker/issues/204#issuecomment-552755422)
 - Polis (POLIS)
 - Viacoin (VIA)
 
 Altcoins are maintained by their respective communities.
 
-## Can an XYZ coin be added in BTCPay?
+## Can XYZ coin be added in BTCPay?
```

Comment:
When you make changes in the tittle or add a new one, you'll have to do it in[ faq-readme ](https://github.com/btcpayserver/btcpayserver-doc/blob/master/FAQ/readme.md)as well, so that links don't break the FAQ ToC page.

Reply:
ah that sucks, I'll undo this